### PR TITLE
feat(pageContentWrapper): L3-3860 Create Wrapper component for Seldon

### PR DESCRIPTION
**Pull request by smarks8**
**Jira ticket**

[L3-3860](https://phillipsauctions.atlassian.net/browse/L3-3860)

**Screenshots**
Screenshot 2024-09-23 092824

**Summary**

- Create wrapper component to have unified padding for content on all Remix pages

**Change List (describe the changes made to the files)**

- Create new component files

- Additions, Changes, and Removals

**Acceptance Test (how to verify the PR)**

- See storybook 

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-3860]: https://phillipsauctions.atlassian.net/browse/L3-3860?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/PageContentWrapper/PageContentWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   */
+  children: React.ReactNode;
+}
+
+/**
+ * ## Overview
+ *
+ * This component wraps content on a page to maintain consistent page padding
+ *
+ * [Figma Link](Add Figma URL here)
```
Comment (scottdickerson) [inline src/components/PageContentWrapper/PageContentWrapper.tsx:19]: I know there's not a component for this in Figma so could you just link to the Lot Details page and put a comment that this page is wrapped in it?  Thanks!

**Comment by scottdickerson** [comment]
@smarks8 I think as part of this we have to remove the left and right padding from the `_row.scss` so it doesn't get double padded once used in a `PageContent`

**Comment by constantinehuzenko** [comment]
LGTM

**Review by scottdickerson — APPROVED** [review]
LGTM!

**Pull request merged**